### PR TITLE
[#64] script: generate ssl certificat

### PR DESCRIPTION
**Pull request by mathben**
- Add submodule dehydrated to generate ssl certificat
- script justletsencrypt.sh to use dehydrated
- argument --ssl will enable https
- support request from let's encrypt to enable ssl cert

### CODE REVIEW

File: src/web/__main__.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def parse_args():
     group.add_argument('-l', '--web-listen-address', dest='listen', default=Listen(), type=parse_listen,
                        help='Web: Web server listen address')
     group.add_argument('--ssl', default=False, action='store_true',
-                       help='Active https and create ssl files if not exist. Not work in windows.')
```
Comment (alexisbssn) [inline src/web/__main__.py]: Activates Https and creates ssl files if they are not present. Does not work in Windows environments.

File: src/web/handlers.py [hunk 2]
```diff
@@ -281,7 +311,9 @@ def get(self):
 class LogoutHandler(base_handler.BaseHandler):
     def get(self):
         if self._global_arg["disable_login"]:
-            return
+            # Bad Request
+            self.set_status(400)
+            raise tornado.web.Finish()
```
Comment (alexisbssn) [inline src/web/handlers.py]: Unauthorized http code is 401

File: src/web/handlers.py [hunk 3]
```diff
@@ -294,20 +326,26 @@ class AdminHandler(base_handler.BaseHandler):
     @tornado.web.authenticated
     def get(self):
         if self._global_arg["disable_admin"]:
-            return
+            # Not Found
+            self.set_status(404)
```
Comment (alexisbssn) [inline src/web/handlers.py]: We would need consistency in status codes. Everyone gets BadRequest for disabled stuff, or 401, 403, 404, or 501. Just stick with a standard

Comment (mathben) [reply src/web/handlers.py]: Done, thanks

**Pull request merged**